layers/+emacs/helpful/README.org

```diff
@@ -0,0 +1,26 @@
+#+TITLE: helpful layer
+
+* Table of Contents                                        :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install[]]
```

Comment:
install link broken and trailing : on the Features link

Reply:
Nice catch on the broken link. The `:` is there for a lot of layers (like Python, for example). It's because the header is `Features:` and the thing will always autocorrect the `:` back in on save.